### PR TITLE
Update getting started with stack edge

### PR DESCRIPTION
**Pull request by Skejven**
Update to the latest `1.5.0-SNAPSHOT` upgrades:
- `forms` instead of `action knot`
- `te` instead of `hbs knot`

### CODE REVIEW

File: src/render/tutorials/getting-started-with-knotx-stack/edge.html.md [hunk 1]
```diff
@@ -104,19 +105,19 @@ fileSystemRepo = knotx.core.repository.filesystem
...
+    Deployed templateEngine=java:io.knotx.te.core.TemplateEngineKnot [606151e2-3971-4732-9eb9-60beff83ef1f]
+    Deployed dataSourceHttp=java:io.knotx.databridge.http.HttpDataSourceAdapter [17a6c160-2421-46d2-aca7-1d7f2e0ca71d]
+    Deployed server=java:io.knotx.server.KnotxServerVerticle [644ca0e5-1df6-49a7-a26f-d2f123f777f2]
+    Deployed fileSystemRepo=java:io.knotx.repository.fs.FilesystemRepositoryConnectorVerticle [0879f874-1276-44b9-b746-585ab19f7d25]
 ```
 
 Now, let's configure `fileSystemRepo` to read files from the local filesystem.
 
-Create `content` directory in `KNOTX_HOME` and put there following page template with Knot.x snippet (`<script data-knotx-knots="databridge,handlebars">...`):
+Create `content` directory in `KNOTX_HOME` and put there following page template with Knot.x snippet (`<script data-knotx-knots="databridge,te">...`):
```
Comment (tomaszmichalak) [inline src/render/tutorials/getting-started-with-knotx-stack/edge.html.md]: knotx:snippet

File: src/render/tutorials/getting-started-with-knotx-stack/edge.html.md [hunk 2]
```diff
@@ -143,7 +144,7 @@ Create `content` directory in `KNOTX_HOME` and put there following page template
     </div>
   </div>
   <div class="row">
-    <script data-knotx-knots="databridge,handlebars"
+    <script data-knotx-knots="databridge,te"
```
Comment (tomaszmichalak) [inline src/render/tutorials/getting-started-with-knotx-stack/edge.html.md]: Please check all snippets tags in this article.

File: src/render/tutorials/getting-started-with-knotx-stack/edge.html.md (hunk 1 above)
Comment (Skejven) [reply src/render/tutorials/getting-started-with-knotx-stack/edge.html.md]: That would require changing default `tagName` to `knotx:snippet` here:
https://github.com/Knotx/knotx-stack/blob/master/knotx-stack-manager/src/main/packaging/conf/application.conf#L30

Comment (Skejven) [reply src/render/tutorials/getting-started-with-knotx-stack/edge.html.md]: Discussed, we will make `knox:snippet` the official default tag in the next major release.

**Pull request merged**